AmplifyTools/AmplifyXcode/Sources/AmplifyXcode/CLICommandReportable.swift

```diff
@@ -17,9 +17,9 @@ extension CLICommandReportable {
         for task in tasks {
             switch task {
             case .success(let message):
-                print("-- ✅ \(message)")
+                print("-- \(message)")
             case .failure(let error):
-                print("-- 🚫 \(error)")
+                print("-- \(error)")
```

Comment:
Updated output according to CLI style (no emoji)

Reply:
Too bad, but it makes sense to conform to CLI style.